app/api/admin_statistics_api/events.py

```diff
@@ -1,42 +1,40 @@
 from flask_rest_jsonapi import ResourceDetail
-from marshmallow_jsonapi.flask import Schema
-from marshmallow_jsonapi import fields
-from datetime import datetime
-import pytz
+from flask import jsonify, Blueprint
+from flask_jwt import jwt_required
+from sqlalchemy.sql import text
 
-from app.api.helpers.utilities import dasherize
 from app.api.bootstrap import api
 from app.models import db
-from app.models.event import Event
 from app.api.data_layers.NoModelLayer import NoModelLayer
-from app.api.helpers.db import get_count
-
-
-class AdminStatisticsEventSchema(Schema):
-    """
-    Api schema
-    """
-    class Meta:
-        """
-        Meta class
-        """
-        type_ = 'admin-statistics-event'
-        self_view = 'v1.admin_statistics_event_detail'
-        inflect = dasherize
-
-    id = fields.String()
-    draft = fields.Method("events_draft_count")
-    published = fields.Method("events_published_count")
-    past = fields.Method("events_past_count")
-
-    def events_draft_count(self, obj):
-        return get_count(Event.query.filter_by(state='draft'))
-
-    def events_published_count(self, obj):
-        return get_count(Event.query.filter_by(state='published'))
-
-    def events_past_count(self, obj):
-        return get_count(Event.query.filter(Event.ends_at < datetime.now(pytz.utc)))
+from app.api.schema.admin_statistics_schema.events import AdminStatisticsEventSchema
+
+event_statistics = Blueprint('event_statistics', __name__, url_prefix='/v1/admin/statistics')
+
+
+@event_statistics.route('/event-topics', methods=['GET'])
+@jwt_required()
+def event_topic_count():
+    result_set = db.engine.execute(text(
+        "SELECT event_topics.name, COUNT(events.id) FROM event_topics " +
+        "LEFT JOIN events ON events.event_topic_id = event_topics.id " +
+        "GROUP BY event_topics.id;"
+    ))
+    event_topics_counts = {}
+    event_topics_counts = dict(list(result_set))
```

Comment:
No need of reassigning

Reply:
Mean removing `event_topics_counts = {}` ?